geopandas/array.py

```diff
@@ -1394,6 +1394,8 @@ def __contains__(self, item):
 
 
 def _get_common_crs(arr_seq):
+    # mask out all None arrays (commonly from concat where column is missing)
+    arr_seq = [ga for ga in arr_seq if not ga.isna().any()]
```

Comment:
Actually, should this be all instead of any?

```suggestion
    arr_seq = [ga for ga in arr_seq if not ga.isna().all()]
```

Reply:
yep, really good catch, would have been bad to miss. I've added a test to explicitly capture that a case with a mix of None and non none entries (without a crs now including the above change) still generates a warning.

(I'm also hoping thes might have some impact on the failures in a bunch of environments, for some reason I'm not having much luck replicating locally, perhaps I need new test environments from scratch)